inspire/modules/workflows/tasks/submission.py

```diff
@@ -287,3 +288,76 @@ def add_files_to_task_results(obj, eng):
         obj.add_task_result(file_obj.name,
                             fileinfo,
                             "workflows/results/files.html")
+
+
+def create_curation_ticket(template, queue="Test", ticket_id_key="ticket_id"):
+    """Create a ticket for the curation.
+
+    Creates the ticket in the HEP_curation queue and stores the ticket ID
+    in the extra_data key specified in ticket_id_key."""
+    @wraps(create_curation_ticket)
+    def _create_curation_ticket(obj, eng):
+        if not obj.extra_data.get("core"):
+            obj.log.info("Curation ticket creation skipped.")
+            return
+        from invenio.modules.access.control import acc_get_user_email
+        from inspire.utils.tickets import get_instance
+        from flask import render_template
+
+        d = Deposition(obj)
+        email = acc_get_user_email(obj.id_user)
+        rt_queue = "Test"
+        sip = d.get_latest_sip(sealed=True)
+
+        recid = sip.metadata.get('recid', None)
+        if recid:
+            recid = '(#{})'.format(recid)
+
+        record_url = obj.extra_data.get('url')
+
+        arxiv_id = sip.metadata.get('arxiv_id')
+        if arxiv_id and is_arxiv_post_2007(arxiv_id):
+            arxiv_id = 'arXiv:' + arxiv_id
+
+        report_number = sip.metadata.get('report_number', None)
+        if report_number:
+            report_number = report_number[0].get('primary')
+
+        link_to_pdf = obj.extra_data.get('submission_data').get('url')
+
+        subject = ' '.join(filter(lambda x: x is not None,
+                           [arxiv_id,
+                            sip.metadata.get('doi'),
+                            report_number,
+                            recid]))
+
+        details = '\n'.join(['{0}: {1}'.format(k.title(), sip.metadata[k]) for k in sip.metadata])
```

Comment:
all the values of metadata are being put into the ticket?? Can you please provide an example of how the ticket will look like?